cli/remotebazel/remotebazel.go

```diff
@@ -538,6 +539,19 @@ func Run(ctx context.Context, opts RunOpts, repoConfig *RepoConfig) (int, error)
 		envVars[envVar] = val
 	}
 
+	execProps := make(map[string]string, 0)
+	for _, prop := range *execPropsFlag {
+		keyValArr := strings.SplitN(prop, "=", 2)
+		if len(keyValArr) != 2 {
+			log.Warnf("Invalid exec property %s. Key value pairs must be separated by '='.", prop)
+			continue
+		}
+		key := keyValArr[0]
+		val := keyValArr[1]
+
+		execProps[key] = val
+	}
```

Comment:
nit: can replace this with `rexec.MakePlatform(*execPropsFlag...)` from `server/util/rexec`